ft-fine-tune-bert/fine-tune-unittests.py

```diff
@@ -1,55 +1,72 @@
+    #         ],
+    #      ]
 
-        with open("6_genre_clean_training_data_small_unittests.txt", "w") as output:
-            for row in self.train_data:
-                for col in row: 
-                    output.write(str(col) + '\n')
+    #     with open("6_genre_clean_training_data_small_unittests.txt", "w") as output:
+    #         for row in self.train_data:
+    #             for col in row: 
+    #                 output.write(str(col) + '\n')
 
-        with open("6_genre_eval_data_small_unittests.txt", "w") as output:
-            for row in self.eval_data:
-                for col in row:
-                    output.write(str(col) + '\n')
+    #     with open("6_genre_eval_data_small_unittests.txt", "w") as output:
+    #         for row in self.eval_data:
+    #             for col in row:
+    #                 output.write(str(col) + '\n')
 
 
     def assertIsFile(self, path):
```

Comment:
This function which you calling in your test cases is raising exceptions.   Hence you need to rewrite test 1 and 2. 

I mentioned before you should not raise exceptions in your testing code.  However, if your application under  test  throwing exceptions then you could test these.